src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -90,17 +90,16 @@ public static class CLIOptions
         public static readonly CLIOption<int> MaxConcurrentTestRunners = new CLIOption<int>
         {
             ArgumentName = "--max-concurrent-test-runners",
-            ArgumentShortName = "-m <integer>",
+            ArgumentShortName = "-c <integer>",
```

Comment:
This is super breaking, we should consider this carefully and I would prefer to do this in a separate pr so we can go over and fix all flags we deem wrong at once.